### PR TITLE
Checkpoint test playbook

### PR DESCRIPTION
**Pull request by shaniacht1**

### CODE REVIEW

File: Scripts/script-CheckpointFWBackupStatus [hunk 1]
```diff
@@ -1,8 +1,8 @@
 commonfields:
-  id: 55f24028-c703-47be-809b-b0774d927b64
-  version: 17
+  id: f3e33e2c-a250-4f45-838c-d71eeb51d57f
```
Comment (aviadl) [inline Scripts/script-CheckpointFWBackupStatus]: id should be like script name

File: Scripts/script-CheckpointFWBackupStatus [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+  version: 15
```
Comment (aviadl) [inline Scripts/script-CheckpointFWBackupStatus]: version = -1

File: Scripts/script-CheckpointFWBackupStatus (hunk 1 above)
Comment (aviadl) [inline Scripts/script-CheckpointFWBackupStatus]: Then make sure to fix the test playbook accordingly

Comment (shaniacht1) [inline Scripts/script-CheckpointFWBackupStatus]: fixed

File: Scripts/script-CheckpointFWBackupStatus (hunk 2 above)
Comment (shaniacht1) [inline Scripts/script-CheckpointFWBackupStatus]: fixed

**Pull request merged**